assets/javascripts/kitten/components/form/text-input-with-unit/index.js

```diff
@@ -130,8 +129,7 @@ export class TextInputWithUnit extends PureComponent {
     return (
       <StyledTextInputWithUnit>
         <StyledTextInput
-          {...others}
-          ref={this.input}
+          ref={input => (this.input = input)}
```

Comment:
```suggestion
          ref={input => { this.input = input }}
```

Reply:
Est-ce qu'on peut pas carrément se passer des parenthèses et des accolades ?

```suggestion
          ref={input => this.input = input}
```

Reply:
Si on vire les accolades, on retourne l'assignation, ce n'est pas vraiment ce que l'on souhaite :). En revanche pour les parenthèses du paramètre carrément 👍